### PR TITLE
Coinbase: fetchTickers & parseTicker methods

### PR DESCRIPTION
**Pull request by ttodua**
- created 'fetchTickers' endpoint
- separated functionality from 'fetchTicker' into 'parseTicker' function (this latter method also server 'fetchTickers' response parsing)

### CODE REVIEW

File: js/coinbase.js [hunk 1]
```diff
@@ -670,6 +670,39 @@ module.exports = class coinbase extends Exchange {
...
+            'currency': this.safeValue (params, 'currency', 'USD'),
+        };
+        const response = await this.publicGetExchangeRates (this.extend (request, params));
+        const data = this.safeValue (response, 'data', {});
+        const rates = this.safeValue (data, 'rates', {});
+        const result = {};
+        const coinKeys = Object.keys (rates);
+        for (let i = 0; i < coinKeys.length; i++) {
+            const coinName = coinKeys[i];
+            const value = rates[coinName]; // already in string, no need for safeNumber
```
Comment (kroitor) [inline js/coinbase.js:686]: This line is a bit wrong – we always need safeNumber or safeString, cause we will switch from float-representations to string-representations everywhere eventually.

File: js/coinbase.js [hunk 2]
```diff
@@ -670,6 +670,39 @@ module.exports = class coinbase extends Exchange {
...
+            const symbol = coinName + '/' + request['currency'];
```
Comment (kroitor) [inline js/coinbase.js:687]: We should look up for the market id with `safeMarket (marketId, market)` here.

File: js/coinbase.js [hunk 3]
```diff
@@ -683,31 +716,43 @@ module.exports = class coinbase extends Exchange {
...
+                'buy': buy,
+                'sell': sell,
+                'spot': spot,
+            },
+        };
+        return this.parseTicker (ticker, market);
+    }
+
+    parseTicker (ticker, market = undefined) {
+        return {
```
Comment (kroitor) [inline js/coinbase.js:735]: Let's always use `this.safeTicker` in return of all `parseTicker` methods (see how this is done in the other exchanges).

**Comment by ttodua** [comment]
Strange thing was that 'datetime' property of `fetchTicker` for coinbase was buggeous till today . it returned me "1970-01..." (probably because timestamp property was 10 digit). Maybe most people havent used it all during all these times, because it wouldn't have been left unnoticed.

**Comment by ttodua** [comment]
@kroitor this is also ready for now, lmk if I had any discrepancy in implementation.

**Pull request merged**